src/graphics/webgl/webgl-graphics-device.js

```diff
@@ -791,6 +791,7 @@ class WebglGraphicsDevice extends GraphicsDevice {
             }
             this.extColorBufferFloat = null;
             this.extDisjointTimerQuery = null;
+            this.extDepthTexture = gl.getExtension('WEBGL_depth_texture');
```

Comment:
Can you give some background why this PR needs this extension?

Reply:
added to the PR description, thanks.